app/presenters/find_this_item_presenter.rb

```diff
@@ -3,13 +3,18 @@
 class FindThisItemPresenter
   attr_reader :document, :config
 
-  def initialize(document:)
+  def initialize(document:, solr_document:)
```

Comment:
Are the `document` and the `solr_document` the same?

Reply:
Yes, I don't believe it should be needed to be added separately. I think you need to add it to the yaml file that the presenter is using... See Rachel K's two recent date PRs for a similar change. We might also want to do a refactor to make adding / changing these views a bit simpler.

Reply:
Nope.